### PR TITLE
[FLINK-22694][e2e] Use sql file in TPCH end to end tests

### PR DESCRIPTION
**Pull request by fsk119**
## What is the purpose of the change

*Use sql files to init environment in TPCH tests.*

### CODE REVIEW

File: flink-end-to-end-tests/test-scripts/test_tpch.sh [hunk 1]
```diff
@@ -51,29 +51,27 @@ ORIGIN_QUERY_DIR="$TARGET_DIR/query"
...
-SOURCES_YAML=$(cat "$TPCH_DATA_DIR/source.yaml")
-SOURCES_YAML=${SOURCES_YAML//\$TABLE_DIR/"$TABLE_DIR"}
+SOURCES_SQL=$(cat "$TPCH_DATA_DIR/source.sql")
+SOURCES_SQL=${SOURCES_SQL//\$TABLE_DIR/"$TABLE_DIR"}
 
 for i in {1..22}
 do
     echo "Running query #$i..."
 
     # First line in sink yaml is ignored
```
Comment (wuchong) [inline flink-end-to-end-tests/test-scripts/test_tpch.sh:65]: update the comment.

File: flink-end-to-end-tests/flink-tpch-test/src/main/java/org/apache/flink/table/tpch/TpchResultComparator.java [hunk 2]
```diff
@@ -81,7 +93,10 @@ public static void main(String[] args) throws IOException {
                                 failed = (e * 0.99 > t || e * 1.01 < t);
                             }
                         } catch (NumberFormatException nfe2) {
-                            failed = !expected[i].trim().equals(actual[i].trim());
+                            failed =
+                                    !expected[i]
+                                            .trim()
+                                            .equals(actual[i].replaceAll("\"", "").trim());
```
Comment (wuchong) [inline flink-end-to-end-tests/flink-tpch-test/src/main/java/org/apache/flink/table/tpch/TpchResultComparator.java:99]: Why there are quotes on numeric?

**Review by wuchong — COMMENTED** [review]
I only have two questions.

File: flink-end-to-end-tests/flink-tpch-test/src/main/java/org/apache/flink/table/tpch/TpchResultComparator.java (hunk 2 above)
Comment (fsk119) [reply flink-end-to-end-tests/flink-tpch-test/src/main/java/org/apache/flink/table/tpch/TpchResultComparator.java:99]: In the new data type system, we will use qoutoes for String type, e.g. "string value", which is different from the behaviour before. Therefore, we remove the quotes when we can't convert the value to `long` and `double`.

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**